coralproject/talk
# [CORL-3072]: Update user preference options for in-page notifications

## What does this PR do?

Updates user preferences for in-page notifications to remove the interface options, and to add in overall enable/disable options.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

Removes interface options, adds in overall `enabled` for User's `inPageNotifications`.

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Go to `My profile` -> `Preferences`. See in-page notifications and enable/disable. Save enabled and disabled. Also see that the `Include notifications when` options are disabled when either in-page notifications are disabled or disabled is selected.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
